### PR TITLE
[FSDE-17745] Implemented code snippets for checkbox webcomponent

### PR DESCRIPTION
**Pull request by soujanreddy**

### CODE REVIEW

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-363/1/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://ff59cdb52959089e5fd0f3315278e29bd6da73be-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-363-ux-chi.kubeodc-test.corp.intranet/ (internal)

File: src/website/views/components/forms/checkbox/_examples.pug [hunk 1]
```diff
@@ -97,12 +103,14 @@ p.-text
...
-        a(href=`#webcomponent-disabled`) Web Component
       li.-active
+        a(href=`#webcomponent-disabled`) Web Component
+      li
         a(href=`#html-disabled`) HTML Blueprint
-  .chi-tabs-panel#webcomponent-disabled
-  .chi-tabs-panel.-active#html-disabled
+  .chi-tabs-panel.-active#webcomponent-disabled
+    :code(lang="html")
+      <chi-checkbox id="checkbox-dis1" label="Checkbox" disabled></chi-checkbox>
```
Comment (josecarlosgonzalezv) [inline src/website/views/components/forms/checkbox/_examples.pug:112]: I would use the same IDs as in the HTLM Blueprint.
```suggestion
      <chi-checkbox id="checkbox-di1" label="Checkbox" disabled></chi-checkbox>
```

File: src/website/views/components/forms/checkbox/_examples.pug [hunk 2]
```diff
@@ -204,12 +224,17 @@ p.-text
...
-        a(href=`#webcomponent-message`) Web Component
       li.-active
+        a(href=`#webcomponent-message`) Web Component
+      li
         a(href=`#html-message`) HTML Blueprint
-  .chi-tabs-panel#webcomponent-message
-  .chi-tabs-panel.-active#html-message
+  .chi-tabs-panel.-active#webcomponent-message
+    :code(lang="html")
+      <chi-checkbox id="checkbox-ms1" label="Checkbox"></chi-checkbox>
```
Comment (josecarlosgonzalezv) [inline src/website/views/components/forms/checkbox/_examples.pug:233]: I would use the same IDs as in the HTLM Blueprint.
```suggestion
      <chi-checkbox id="checkbox-me1" label="Checkbox"></chi-checkbox>
```

File: src/website/views/components/forms/checkbox/_examples.pug [hunk 3]
```diff
@@ -316,12 +355,78 @@ p.-text
...
+          </div>
+        </div>
+        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
+          <div class="chi-form__item">
+            <chi-checkbox id="checkbox-gr11" label="Option 11"></chi-checkbox>
+          </div>
+        </div>
+        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
+          <div class="chi-form__item">
+            <chi-checkbox id="checkbox-gr12" label="Option 12"></chi-checkbox>
```
Comment (josecarlosgonzalezv) [inline src/website/views/components/forms/checkbox/_examples.pug:425]:
```suggestion
            <chi-checkbox id="checkbox-g1" label="Option 1"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g2" label="Option 2"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g3" label="Option 3"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g4" label="Option 4"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g5" label="Option 5"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g6" label="Option 6"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g7" label="Option 7"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g8" label="Option 8"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g9" label="Option 9"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g10" label="Option 10"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g11" label="Option 11"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g12" label="Option 12"></chi-checkbox>
```

Comment (josecarlosgonzalezv) [inline src/website/views/components/forms/checkbox/_examples.pug:425]: I would use the same IDs as in the HTLM Blueprint.
```suggestion
            <chi-checkbox id="checkbox-g1" label="Option 1"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g2" label="Option 2"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g3" label="Option 3"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g4" label="Option 4"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g5" label="Option 5"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g6" label="Option 6"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g7" label="Option 7"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g8" label="Option 8"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g9" label="Option 9"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g10" label="Option 10"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g11" label="Option 11"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g12" label="Option 12"></chi-checkbox>
```

Comment (josecarlosgonzalezv) [inline src/website/views/components/forms/checkbox/_examples.pug:425]: I would use the same IDs as in the HTLM Blueprint.
```suggestion
            <chi-checkbox id="checkbox-g1" label="Option 1"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g2" label="Option 2"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g3" label="Option 3"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g4" label="Option 4"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g5" label="Option 5"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g6" label="Option 6"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g7" label="Option 7"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g8" label="Option 8"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g9" label="Option 9"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g10" label="Option 10"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g11" label="Option 11"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g12" label="Option 12"></chi-checkbox>
```

Comment (josecarlosgonzalezv) [inline src/website/views/components/forms/checkbox/_examples.pug:425]: I would use the same IDs as in the HTLM Blueprint.
```suggestion
            <chi-checkbox id="checkbox-g1" label="Option 1"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g2" label="Option 2"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g3" label="Option 3"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g4" label="Option 4"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g5" label="Option 5"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g6" label="Option 6"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g7" label="Option 7"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g8" label="Option 8"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g9" label="Option 9"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g10" label="Option 10"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g11" label="Option 11"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g12" label="Option 12"></chi-checkbox>
```

Comment (josecarlosgonzalezv) [inline src/website/views/components/forms/checkbox/_examples.pug:425]: I would use the same IDs as in the HTLM Blueprint.
```suggestion
            <chi-checkbox id="checkbox-g1" label="Option 1"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g2" label="Option 2"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g3" label="Option 3"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g4" label="Option 4"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g5" label="Option 5"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g6" label="Option 6"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g7" label="Option 7"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g8" label="Option 8"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g9" label="Option 9"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g10" label="Option 10"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g11" label="Option 11"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g12" label="Option 12"></chi-checkbox>
```

Comment (josecarlosgonzalezv) [inline src/website/views/components/forms/checkbox/_examples.pug:425]: I would use the same IDs as in the HTLM Blueprint.
```suggestion
            <chi-checkbox id="checkbox-g1" label="Option 1"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g2" label="Option 2"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g3" label="Option 3"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g4" label="Option 4"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g5" label="Option 5"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g6" label="Option 6"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g7" label="Option 7"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g8" label="Option 8"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g9" label="Option 9"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g10" label="Option 10"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g11" label="Option 11"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g12" label="Option 12"></chi-checkbox>
```

**Review by josecarlosgonzalezv — CHANGES_REQUESTED** [review]

File: src/website/views/components/forms/checkbox/_examples.pug (hunk 3 above)
Comment (josecarlosgonzalezv) [inline src/website/views/components/forms/checkbox/_examples.pug:425]: I would use the same IDs as in the HTLM Blueprint.
```suggestion
            <chi-checkbox id="checkbox-g1" label="Option 1"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g2" label="Option 2"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g3" label="Option 3"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g4" label="Option 4"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g5" label="Option 5"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g6" label="Option 6"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g7" label="Option 7"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g8" label="Option 8"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g9" label="Option 9"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g10" label="Option 10"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g11" label="Option 11"></chi-checkbox>
          </div>
        </div>
        <div class="chi-col -w--12 -w-md--3 -w-sm--6 -mb--1">
          <div class="chi-form__item">
            <chi-checkbox id="checkbox-g12" label="Option 12"></chi-checkbox>
```

**Review by josecarlosgonzalezv — CHANGES_REQUESTED** [review]

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-363/2/. :white_check_mark:

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-363/3/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://a47447cf3e4e23c079b4027170dba7f2e7ef0b6c-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-363-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Comment by soujanreddy** [comment]
@jllr This PR is not getting updated with my new commits for some reason. Closing this and reopening a new PR with the updated changes.

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-363/4/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://507c55e19eae9bc59c674e7dbf0574fadc90e518-chi-pr.enterprise.ctl.io (external) and in https://nginx-pr-363-ux-chi.kubeodc-test.corp.intranet/ (internal)

**Pull request closed without merging**